### DISCUSSION TITLE
Rotary encoder being very unstable

### DISCUSSION THREAD

**Discussion by LovelyA72** [probonopd/MiniDexed#362] [Q&A] [answered]
I am currently using the rotary encoder shown in the README.md

But when I rotate the encoder instead of one click one move, I need to rotate quite a few clicks before a successful move

**Comment by BobanSpasic** [comment, marked as answer]
Please, search the Discussion topics - you need to put some capacitors between the contacts of the encoder.
You will find in discussions what are the experiences with different values for the capacitors.
One from the topics: https://github.com/probonopd/MiniDexed/discussions/322
